README_DOCKER.md

```diff
@@ -62,3 +62,11 @@ docker exec -t -i $(docker ps | grep "salt-api" | cut -d " " -f 1) bash
 ```
 docker-compose run
 ```
+
+# Saltpad with docker-compose and salt-api stable
+
+There is also an alternative Docker image that use salt-api stable instead of develop version.
```

Comment:
There is an alternative Docker image that use salt-api stable instead of develop version.